internal/cmd/config/set/set.go

```diff
@@ -70,7 +71,7 @@ func NewCmd() *cobra.Command {
 			}
 
 			if model.SessionTimeLimit != nil {
-				cmd.Println("Authenticate again to apply changes to session time limit")
+				p.Info("Authenticate again to apply changes to session time limit")
```

Comment:
This is rather a warning, you can leave it for the PR to replace the warnings to be more clear